### ISSUE TITLE
Filedialog unreadable

### ISSUE THREAD

**Issue by AnjaRy** [rdbende/Forest-ttk-theme#16]
I use this theme for an application on Linux. 
In both, dark and light mode, it is impossible to read the filenames in the filedialog boxes, as they are white foreground on white background. After selection, they appear black.
Is there an easy solution to this I can use?

![image](https://github.com/user-attachments/assets/08d869e3-c1c1-488d-8280-d194676cb11a)

**Comment by rdbende** [comment]
Unfortunately I couldn't figure out how to fix this, even after a couple of years.

Here are similar issues in my other repositories for reference:
https://github.com/rdbende/Sun-Valley-ttk-theme/issues/104
https://github.com/rdbende/Azure-ttk-theme/issues/53
https://github.com/rdbende/Azure-ttk-theme/issues/35

**Comment by Dakyz** [comment]
u can try add black foreground after theme init:
```Python
# Set the initial theme
self.root.tk.call('source', 'Gui/Theme/forest-dark.tcl')
ttk.Style().theme_use('forest-dark')

# add this
self.root.option_add('*foreground', 'black')
```

or just use light theme)

**Comment by AnjaRy** [comment]
> u can try add black foreground after theme init:

Thanks for the suggestion, this would solve the problem in the filedialogbox; However, it makes more problems, as now the menulist and some other elements are not readable anymore....

Is there an option to change the background of the filedialogbox?

> or just use light theme)

I want to offer both options to the users in the end, so this does not really solve my problem

**Comment by Dakyz** [comment]
> However, it makes more problems, as now the menulist and some other elements are not readable anymore....

yes, I know. but I would say at least more readable in menulist than in the filedialog boxes previously.

> Is there an option to change the background of the filedialogbox?

I don't know...
